### ISSUE TITLE
Deauthing attack problem

### ISSUE THREAD

**Issue by GUIRAXRR** [aircrack-ng/mdk4#44]
Trying to perform a deauth attack on my own AP but no clients are being disconnected from it. Any idea what could be wrong?

**Comment by vonuyvico** [comment]
Yeah me too, I was deauthing a specific BSSID. It is sending 1 packet per second

**Comment by E7mer** [comment]
Could you be more in detail？
